data/campaigns/Eastern_Invasion/scenarios/11_Captured.cfg

```diff
@@ -1135,100 +1357,144 @@
-                speaker=Gweddry
-                message= _ "They have reinforcements pouring in from the eastern entrance! We have to get out of here quickly!"
-            [/message]
+                    [then]
+                        [message]
+                            speaker=Addogin
+                            message= _ "Guess we’re on the run again! My mercenary boys back home woulda made short work o’ these Whitefang punks... I sure am missin’ the Estmarks right about now."
+                        [/message]
+                    [/then]
 
-            [place_shroud]
-                side=1
-                x,y=41,15
-                radius=2
-            [/place_shroud]
+                    [else]
+                        [message]
+                            speaker=Gweddry
+                            message= _ "Everyone hurry, we have to get out of here!"
+                        [/message]
+                    [/else]
+                [/if]
+
+                [place_shroud]
+                    side,x,y,radius=1,46,11,2
+                [/place_shroud]
+                [place_shroud]
+                    side,x,y,radius=1,46,25,2
+                [/place_shroud]
+
+                # foreshadow the drowned plains
+                [event]
+                    name=new turn
+                    [event]
+                        name=side 1 turn end
+                        [message]
+                            speaker=Owaec
+                            message= _ "Such carnage in this dark confined cave... how I long to once again gaze upon the rolling plains of my homeland. These lands are cold, but the thought of my fellow clansmen is as a fire burning in my breast."
+                        [/message]
+                        [message]
+                            speaker=Gweddry
+                            message= _ "Wesnoth’s best horses and much of Weldyn’s food is come from there. I’m sure it’s well defended, despite never having visited there myself."
+                        [/message]
+                        [message]
+                            speaker=Owaec
+                            message= _ "There is naught that I can speak that would truly do them justice! You will have to see the plains yourself to appreciate their majesty."
+                        [/message]
+                        [message]
+                            speaker=Gweddry
+                            message= _ "Then let us return quickly to Wesnoth to aid in their defense!"
+                        [/message]
+                    [/event]
+                [/event]
+            [/event]
         [/event]
     [/event]
 
     # This event should be fired with the disguised unit as the primary unit, even
     # if indirectly triggered by a name=exposed event on another unit.
     [event]
         name=remove disguise
+        [message]
+            speaker=Chief Dra-Nak
+            message= _ "Hey, who are you? Boja, sic ‘em!"
```

Comment:
Wrong 'curly' quote.

Reply:
Maybe just find and replace all `‘em` with `’em`.